### PR TITLE
Seo tags for google search

### LINKED ISSUE #863: Add meta tags for Search engine optimization

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #863

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
@publiclab/is-reviewers check this.

File: examples/index.html [hunk 1]
```diff
@@ -10,6 +10,12 @@
   <meta name="theme-color" content="#428bca">
   <link rel="icon" sizes="192x192" href="../icons/ic_192.png">
   <link rel="manifest" href="manifest.json">
+   <!--Adding meta Tag for search engine optimisation-->
+   <meta property="og:description" content="A pure JavaScript sequential image processing system, inspired by storyboards. Instead of modifying the original image, it creates a new image at each step in a sequence.">
+   <meta property="og:site_name" content="sequencer.publiclab.org">
+   <meta property="og:url" content="http://sequencer.publiclab.org">
```
Comment (HarshKhandeparkar) [inline examples/index.html]: Add https

File: examples/index.html [hunk 2]
```diff
@@ -10,6 +10,12 @@
...
+   <meta property="og:title" content="image-Sequencer">
```
Comment (HarshKhandeparkar) [inline examples/index.html]: You can have spaces, also please capitalize "Image Sequencer"

**Comment by Harshithpabbati** [comment]
Done @HarshKhandeparkar

File: examples/index.html [hunk 3]
```diff
@@ -10,6 +10,12 @@
   <meta name="theme-color" content="#428bca">
   <link rel="icon" sizes="192x192" href="../icons/ic_192.png">
   <link rel="manifest" href="manifest.json">
+   <!--Adding meta Tag for search engine optimisation-->
+   <meta property="og:description" content="A pure JavaScript sequential image processing system, inspired by storyboards. Instead of modifying the original image, it creates a new image at each step in a sequence.">
+   <meta property="og:site_name" content="http://sequencer.publiclab.org">
```
Comment (HarshKhandeparkar) [inline examples/index.html]: This was correct already! Remove the http://

File: examples/index.html (hunk 1 above)
Comment (HarshKhandeparkar) [reply examples/index.html]: I meant that you should replace http:// with https:// on this line 16

File: examples/index.html [hunk 4]
```diff
@@ -12,8 +12,8 @@
   <link rel="manifest" href="manifest.json">
    <!--Adding meta Tag for search engine optimisation-->
    <meta property="og:description" content="A pure JavaScript sequential image processing system, inspired by storyboards. Instead of modifying the original image, it creates a new image at each step in a sequence.">
-   <meta property="og:site_name" content="http://sequencer.publiclab.org">
-   <meta property="og:url" content="http://sequencer.publiclab.org">
```
Comment (HarshKhandeparkar) [inline examples/index.html]: Oh you left the ://

Comment (Harshithpabbati) [reply examples/index.html]: What???

**Comment by Harshithpabbati** [comment]
@jywarren review this.

**Comment by jywarren** [comment]
:-) nice!

**Comment by jywarren** [comment]
Thanks! Hey, @Harshithpabbati do you think you could help Nirav out on this? https://github.com/publiclab/image-sequencer/pull/869 I'd really appreciate it!

**Comment by Harshithpabbati** [comment]
Sure @jywarren i will

**Pull request merged**